snake.py

```diff
@@ -1,34 +1,41 @@
 import math
 import random
 import pygame
-from pygame import *

```

Comment:
you already imported pygame in the line above. import pygame will allow you to use any pygame function as pygame.<function>. from pygame import * will allow you to use any pygame function as <function> without the pygame.

For most modules, it's best practice to use the style in line 3

Reply:
Thank you, when would you use from x import * rather than import x if theyre functionally the same?